src/components/material/material-buttons/online/MaterialButtonsOnline.tsx

```diff
@@ -45,8 +46,12 @@ const MaterialButtonsOnline: FC<MaterialButtonsOnlineProps> = ({
       url: externalUrl,
       loginRequired
     } = accessElement as AccessUrl;
-    // TODO:  We have experienced that externalUrl is not always valid.
-    // We should use isUrlValid helper function to check is it is valid.
+
+    //  We have experienced that externalUrl is not always valid.
+    if (!isUrlValid(externalUrl)) {
+      return null;
```

Comment:
I agree that not rendering anything is the right approach if the url is invalid but it can be hard to debug if there are situations where buttons do not appear when expected. Consider logging a message to `console.debug` if this occurs. Logging is not something that we do often throughout this project though.

Reply:
We now use error boundary that Mikkel introduced to not break the app but log the info in the console :)